bunq/sdk/context.py

```diff
@@ -254,33 +254,46 @@ def session_context(self):
 
         return self._session_context
 
-    def save(self, path=None):
+    def save(self, path=None, to_json=False):
```

Comment:
>  Functions should do one thing. They should do it well. They should do it only.
>
> -- Uncle Bob

Please make a separate public method (say, `to_json`) and use it in `save`.

(also, though not relevant with the change I suggest: `Union` was introduced in Python 3.5, but we support Python 2 as well, so we do not want it)